yarn.lock

```diff
@@ -5774,9 +5774,9 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@metamask/keyring-api@npm:^10.1.0":
-  version: 10.1.0
-  resolution: "@metamask/keyring-api@npm:10.1.0"
+"@metamask/keyring-api@npm:@metamask-previews/keyring-api@9.0.0-f9cd26b":
```

Comment:
The `yarn.lock` is probably not up-to-date, we should not have any preview builds here.